Core/OxidEeEvents.php

```diff
@@ -28,6 +29,7 @@ class OxidEeEvents
     const ORDER_TABLE = "oxorder";
     const PAYMENT_TABLE = "oxpayments";
     const TRANSACTION_TABLE = "wdoxidee_ordertransactions";
+    const SEPA_DD_PAYMENT_ID = "wdsepadd";
```

Comment:
You can also use `SepaDirectDebitPaymentMethod::getName(true)` to get this value so that you don't have to hardcode it here.